components/home/modules/ROOT/pages/index.adoc

```diff
@@ -14,6 +14,8 @@ Comprehensive guides for every step of your development journey.
 
 include::learn::partial$cards.adoc[]
 
+Check out https://forum.openzeppelin.com/c/general/guides-and-tutorials/23[the OpenZeppelin forum] for additional community-contributed guides!
```

Comment:
```suggestion
Check out https://forum.openzeppelin.com/c/general/guides-and-tutorials[the OpenZeppelin forum] for additional community-contributed guides!
```

Reply:
I wouldn't make that change. Including the ID in the URL ensures that it will work even if we rename the section. For instance, https://forum.openzeppelin.com/c/general/foo/23 works fine.